crates/repl/src/session.rs

```diff
@@ -242,6 +243,34 @@ impl Session {
                         this.update(&mut cx, |session, cx| {
                             // At this point we can create a new kind of kernel that has the process and our long running background tasks
 
+                            let stderr = kernel.process.stderr.take();
+
+                            cx.spawn(|_session, mut _cx| async move {
+                                if let None = stderr {
+                                    return;
+                                }
+                                let reader = BufReader::new(stderr.unwrap());
+                                let mut lines = reader.lines();
+                                while let Some(Ok(line)) = lines.next().await {
+                                    log::error!("kernel: {}", line);
+                                }
+                            })
+                            .detach();
+
+                            let stdout = kernel.process.stderr.take();
```

Comment:
```diff
- let stdout = kernel.process.stderr.take();
+ let stdout = kernel.process.stdout.take();
```

Reply:
Took care of this in https://github.com/zed-industries/zed/pull/15828